File: AppHelloWorld/app/src/main/java/com/example/apphelloworld/MainActivity.java
package com.example.apphelloworld;

import android.os.Bundle;
import android.util.Log;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
            setContentView(R.layout.activity_main);

        try {

            //Log.d("MainActivity", String.valueOf(5/0));
            Log.d("MainActivity", "Happy Birthday to you!");

        } catch (Exception e) {

            Log.e("MainActivity", e.toString());

        }

    }
}

File: AppSorteio/app/src/main/java/com/example/appsorteio/MainActivity.kt
package com.example.appsorteio

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import kotlinx.android.synthetic.main.activity_main.*
import kotlin.random.Random

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun sortearNumero(view: View) {

        val number = Random.nextInt(10);
        textoSorteio.text = "Número sorteado é $number";

    }

}